src/main/java/org/cru/godtools/domain/packages/PageStructure.java

```diff
@@ -224,7 +227,26 @@ else if(!XmlUtilities.nodesMatch(baseNextSibling, updatedNextSibling))
 		return;
 	}
 
-
+	private boolean hasSameAttributes(Element oElement, Element aElement)
```

Comment:
This method now exists statically in `XMLUtilities`.  Can you update that? @LaelWatkins